packages/search/addon/services/navi-search/navi-base-search-provider.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Base search provider service.
+ */
+
+import Service, { inject as service } from '@ember/service';
+import { assert } from '@ember/debug';
+
+export default class NaviBaseSearchProviderService extends Service {
+  /**
+   * @property {Ember.Service} store
+   */
+  @service store;
+
+  /**
+   * @property {String} name
+   */
+  name = undefined;
+
+  /**
+   * @property {String} niceName
+   */
+  niceName = undefined;
```

Comment:
How is this going to be different from name?

Reply:
it's the same name but written not in camel case.

Reply:
Is it necessary to store another property for that or can we just use a string formatter of some sort whenever the nice name is needed?